src/pages.js

```diff
@@ -492,101 +502,109 @@ class Pages extends LitElement
+          ${this.isSidebar && this.collInfo.lists.length ? html`
+            <div class="is-hidden-tablet mobile-lists">
+              <span class="is-size-7">Filter By List:</span>
+              <div class="select is-small">
+                <select id="sort-select" @change=${this.onSelectListDrop}>
+                <option value="0" ?selected="${this.currList === 0}">All Pages</option>
+                ${this.collInfo.lists.map(list => html`
+                  <option value="${list.id}" ?selected="${this.currList === list.id}">${list.title}</option>
+                `)}
+                </select>
+              </div>
+            </div>` : ``}
         </div>
-        <div class="column main-content">
-          ${this.renderPages()}
+
+        <div class="${this.isSidebar ? "sidebar" : "full"}">
+          <div class="main columns">
+
+            <!-- Hidden if the page list is a sidebar on the replay page -->
+            <aside class="column index-bar is-one-fifth is-hidden-mobile" aria-label="Actions">
+              <div class="index-bar-title">${this.collInfo.title}</div>
+
+              <span class="num-results" aria-live="polite" aria-atomic="true">${this.formatResults()}</span>
+
+              ${this.editable ? html`
+              <div class="index-bar-actions">
+                ${this.renderDownloadMenu()}
+              </div>` : ``}
+              ${this.collInfo.lists.length ? html`
+              <p id="filter-label" class="menu-label">Filter By List:</p>
+              <div class="index-bar-menu menu">
+                <ul class="menu-list">
+                  <li>
+                    <a href="#list-0" data-list="0" class="${currList === 0 ? 'is-active' : ''}"
+                      @click=${this.onSelectList}><i>All Pages</i></a>
+                  </li>
+                  ${this.collInfo.lists.map(list => html`
+                    <li>
+                      <a @click=${this.onSelectList} href="#list-${list.id}"
+                      data-list="${list.id}"
+                      class="${currList === list.id ? 'is-active' : ''}">${list.title}</a>
+                    </li>`)}
+                </ul>
+              </div>
+              ` : ``}
+            </aside>
+            <div class="column main-content">
+              <div id="page-list-heading" class="is-sr-only" role="heading" aria-level="${this.isSidebar ? "3": "2"}">Page List</div>
+              ${this.renderPages()}
+            </div>
+          </div>
         </div>
-      </div>
-    </div>`;
+      </div>`;
   }
 
   renderDownloadMenu() {
     return html`
       <div class="dropdown ${this.menuActive ? 'is-active' : ''}">
         <div class="dropdown-trigger">
-          <button @click="${this.onMenu}" class="button is-small" aria-haspopup="true" aria-controls="dropdown-menu">
+          <button @click="${this.onMenu}" class="button is-small" aria-haspopup="true" aria-expanded="${this.menuActive}" aria-controls="dropdown-menu">
```

Comment:
should this be `?aria-expanded`. I remember I checked, but not why it's like this...